### PR TITLE
Add 43907/54907 0.5.0 release

### PR DESCRIPTION
**Pull request by kyle-cypress**
- Add TARGET_CYW943907AEVAL1F v0.5.0
- Add TARGET_CYW954907AEVAL1F v0.5.0

### CODE REVIEW

File: mtb-bsp-manifest-fv2.xml [hunk 1]
```diff
@@ -1,4 +1,84 @@
 <boards>
+  <board>
+    <id>CYW943907AEVAL1F</id>
+    <category>AIROC&#8482; Connectivity BSPs</category>
+    <board_uri>http://git-ore.aus.cypress.com/repo-staging/TARGET_CYW943907AEVAL1F</board_uri>
```
Comment (morser499) [inline mtb-bsp-manifest-fv2.xml]: This URL is wrong

File: mtb-bsp-manifest-fv2.xml [hunk 2]
```diff
@@ -1,4 +1,84 @@
...
+      <version tools_min_version="2.4.0" flow_version="1.0,2.0" prov_capabilities_per_version="bsp_gen3">
+        <num>0.5.0 release</num>
+        <commit>release-v0.5.0</commit>
+      </version>
+    </versions>
+  </board>
+  <board>
+    <id>CYW954907AEVAL1F</id>
+    <category>AIROC&#8482; Connectivity BSPs</category>
+    <board_uri>http://git-ore.aus.cypress.com/repo-staging/TARGET_CYW954907AEVAL1F</board_uri>
```
Comment (morser499) [inline mtb-bsp-manifest-fv2.xml]: URL is wrong

**Review by morser499 — CHANGES_REQUESTED** [review]

File: mtb-bsp-manifest-fv2.xml (hunk 1 above)
Comment (kyle-cypress) [reply mtb-bsp-manifest-fv2.xml]: Fixed

File: mtb-bsp-manifest-fv2.xml (hunk 2 above)
Comment (kyle-cypress) [reply mtb-bsp-manifest-fv2.xml]: Fixed

**Review by morser499 — APPROVED** [review]

**Pull request merged**